tests/test_frame.py

```diff
@@ -728,6 +728,7 @@ def gethead(s: pd.Series, y: int) -> pd.Series:
     )
 
 
+# Deprecated in pandas 2.1.0
```

Comment:
We can now remove this test because we don't test deprecated methods (and the type checking in our CI will fail once you delete `applymap()` from `frame.pyi`

Reply:
Removed applymap